src/FluentMigrator.Runner/VersionLoader.cs

```diff
@@ -147,9 +147,23 @@ public void LoadVersionInfo()
             if (!AlreadyCreatedVersionTable) return;
 
             var dataSet = _processor.ReadTableData(VersionTableMetaData.SchemaName, VersionTableMetaData.TableName);
-            foreach (DataRow row in dataSet.Tables[0].Rows)
+            var dataTable = dataSet.Tables[0];
+
+            // Ignore case when looking for correct column.
+            foreach (DataColumn col in dataTable.Columns)
```

Comment:
This change is not optimal. It might be easier to just set `dataTable.CaseSensitive` to `true`

Reply:
> setting caseSensitive = false didn't work.

Uh, ok. But it still looks strange. Just get the column index and use that instead - but only if `Version` couldn't be found.

Reply:
And please use `OrdinalIgnoreCase`.

Reply:
I'm trying to find better solution.